layouts/system76/gaze16-3050/default.json

```diff
@@ -0,0 +1,425 @@
+{
+  "model": "system76/gaze15",
```

Comment:
This line needs to be changed.

The `default.json` files are generated by `layouts.py` from the `EC` source code. But I think this should be correct since the default layout shouldn't really be changed, other than `"model"`.

Reply:
I'll update that for every model! Sorry!